.github/workflows/tox.yml

```diff
@@ -4,7 +4,8 @@ on:  # yamllint disable-line rule:truthy
   - pull_request
   - push
 env:
-  TOX_LSR: "git+https://github.com/linux-system-roles/tox-lsr@main"
+  TOX_LSR: "git+https://github.com/linux-system-roles/tox-lsr@2.4.0"
```

Comment:
@richm, the other day, you commented as follows... Have we changed our mind?
https://github.com/linux-system-roles/template/pull/63#issuecomment-777595960

Reply:
> @richm, the other day, you commented as follows... Have we changed our mind?
> [#63 (comment)](https://github.com/linux-system-roles/template/pull/63#issuecomment-777595960)

good catch